### PR TITLE
hommework4

### PR DESCRIPTION
**Pull request by Lusine89**

### CODE REVIEW

File: js/lesson_4/src/task.ts [hunk 1]
```diff
@@ -22,13 +24,12 @@ export const users: PersonType[] = [
     {
         name: 'Batman',
         age: 'unknown',
-        isSuperHero: true,
```
Comment (vladimirchizhenkov) [inline js/lesson_4/src/task.ts:25]: Why did you remove this property?
It is optional, same as 'occupation'.

File: js/lesson_4/src/task.ts [hunk 2]
```diff
@@ -5,6 +5,8 @@
 // 4. Вывести значение константы в тег <p>, по аналогии с уже имеющимися примерами;
 
 type PersonType = {
+    name:string;
+    age:number | string;
     occupation?: string;
 };
```
Comment (vladimirchizhenkov) [inline js/lesson_4/src/task.ts:12]: Missing property 'isSuperHero'

File: js/lesson_4/src/task.ts (hunk 1 above)
Comment (Lusine89) [reply js/lesson_4/src/task.ts:25]: I´m already fixed

**Review by vladimirchizhenkov — APPROVED** [review]
